schema.sql

```diff
@@ -1,10 +1,60 @@
 /* Database schema to keep the structure of entire database. */
 
+DROP TABLE IF EXISTS animals;
+
 CREATE TABLE animals (
     id int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     name char(50),
     date_of_birth date,
     escape_attempts int,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species char(50)
 );
+
+
+
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before tranasction.
+
+BEGIN;
+SELECT * FROM animals;
+UPDATE animals SET species = 'unspecified';
+SELECT name, species FROM animals;
+ROLLBACK;
+
+-- Inside a transaction:
+    -- Update the animals table by setting the species column to digimon for all     animals that have a name ending in mon.
+BEGIN;
+UPDATE animals SET species ='digimon' WHERE name LIKE '%mon';
+
+    -- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species = ' ';
+    -- Commit the transaction.
+COMMIT;
+
```

Comment:
* I suggest you transfer all these transactions into the `queries.sql` file as stated in the project requirement [Refer here to confirm](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Remember%20to%20add%20these%20queries%20to%20your%20queries.sql%20file.)